pkg/collector/corechecks/cluster/kubernetes_state_aggregators.go

```diff
@@ -0,0 +1,109 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
We're having multiple `kubernetes_state*.go` files under inside `cluster` package in addition to the kubernetes api server check files. I'm thinking we should organise each check in a separate sub-package. Feel free to ignore if you think it's out of scope.

Reply:
FYI we're going to have even more core checks under `cluster` with https://github.com/DataDog/datadog-agent/pull/7403

Reply:
Indeed, I think that, for consistency, it would make sense to put every core check in its dedicated subpackage and not only the `kubernetes_state` one.
This is big enough and unrelated to this change to deserve a dedicated PR: #7433.

Reply:
thank you 🙇